virttest/utils_test/qemu/__init__.py

```diff
@@ -186,6 +186,32 @@ def clear_win_driver_verifier(driver, vm, timeout=300):
         session.close()
 
 
+@error_context.context_aware
+def check_win_driver(session, driver, timeout=300):
+    """
+    Check if driver is installed and state is running for windows guest.
+    :param session: VM session
+    :param driver: The driver which needs enable the verifier.
+    :param timeout: Timeout in seconds.
+    :return: A tuple (status, output) where status is the driver check result
+    and output is the detail information.
+    """
+    error_context.context("Check %s driver state." % driver,
+                          logging.info)
+    msinfo_cmd = "start /wait msinfo32 /report c:\msinfo.txt"
+    driver_check_cmd = "type c:\msinfo.txt |findstr /i \"%s.sys\" " \
+                       "|findstr /i \"running\"" % driver
+    end_time = time.time() + timeout
+    while time.time() < end_time:
+        session.cmd(msinfo_cmd, timeout)
+        status, output = session.cmd_status_output(driver_check_cmd)
+        if status:
+            time.sleep(4)
+        else:
+            break
+    return (status, output)
```

Comment:
When driver is not installed,the output is always empty.
I saw win_irq_check and pvpanic cases both call this function and print the output msg.
 - https://github.com/autotest/tp-qemu/pull/1134
 - https://github.com/autotest/tp-qemu/pull/1119
If this is  not what you expect,please adjust it.
Thanks

Reply:
It's what I expect, from the output we can see if the driver is installed and if the driver is in stopped status.
Can't adjust it in this function, as it is only check driver's status.  How to use it should be adjusted by the call function.

Thanks.